### PR TITLE
Reorder reference

### PR DESCRIPTION
**Pull request by Asmatzaile**

### CODE REVIEW

File: astro.config.mjs [hunk 1]
```diff
@@ -94,26 +94,21 @@ export default defineConfig({
         { label: 'Reference',
           // translations: { 'fr': 'Référence', },
           items: [
-            { label: 'Tidal functions',
-              autogenerate: { directory: 'reference/tidal' },
-              collapsed: true,
-              badge: 'WIP',
+            { label: 'Tidal',
+              items: [
+                {label: 'Configuration | The boot file', link: 'reference/tidal/configuration' },
```
Comment (trespaul) [inline astro.config.mjs:99]: Can we keep it with a colon, like `Configuration: the boot file` or just `Configuration`?

File: astro.config.mjs [hunk 2]
```diff
@@ -94,26 +94,21 @@ export default defineConfig({
...
-            },
-            { label: 'Configuration',
               items: [
-                { label: 'Tidal', link: '/reference/config/tidal' },
-                { label: 'SuperDirt', link: '/reference/config/superdirt' },
-              ]
-            },
-            { label: 'Mini-notation',
-              autogenerate: { directory: '/reference/mini-notation' },
+                {label: 'Configuration | The startup file', link: 'reference/superdirt/configuration'},
```
Comment (trespaul) [inline astro.config.mjs:106]: similar.

File: astro.config.mjs [hunk 3]
```diff
@@ -94,26 +94,21 @@ export default defineConfig({
...
+                {label: 'Default assets', autogenerate: { directory: 'reference/superdirt/default-assets' }, collapsed: true},
```
Comment (trespaul) [inline astro.config.mjs:107]: Not sure if "assets" is the right word for synths and effects ... but I can't think of anything else. "Default components" maybe? But "assets" works anyway.

**Review by trespaul — CHANGES_REQUESTED** [review]

File: astro.config.mjs (hunk 3 above)
Comment (Asmatzaile) [reply astro.config.mjs:107]: How about 'default library'?

**Review by trespaul — APPROVED** [review]

**Pull request merged**